data-es6.js

```diff
@@ -5323,6 +5323,11 @@ exports.tests = [
         typescript1: typescript.corejs,
         firefox2: false,
         firefox4: true,
+        ie11: {
+          val: true,
```

Comment:
Rather `false`. Also, `.subarray` test depends on `Uint8ClampedArray`.

Reply:
Is this value supposed to reflect initial release or the version that people are probably using?  I believe it is the later.

Reply:
Do you mean the note should be added to .subarray too?  Wouldn't that be confusing?  Maybe the test should be changed to avoid the dependency.

Reply:
For IE, it’s the latest in that major line.

Reply:
> Is this value supposed to reflect initial release or the version that people are probably using? I believe it is the later.

Just an example: we will set is as `true`. `babel-preset-env` will load this result. For IE11 on `Uint8ClampedArray` `preset-env` will not add a polyfill. But a serious part of users has un-updated versions of IE11.

> Do you mean the note should be added to .subarray too?

I don't care, but with current tests it should be `false`.

> Maybe the test should be changed to avoid the dependency.

IIRC it's already was proposed in another issues.

Reply:
Since the compat table assumes all IE users are on the latest of their major line, that sounds like a flaw in preset-env.

Reply:
@ljharb this approach should work for compilers and unstable releases. For stable engines, we should use data for initial releases, otherwise, developers could have similar problems even without `preset-env`.

Reply:
Then we’d need to keep data for multiple ie 11 builds, but also, preset-env would need to provide a way for uses to specify that. Do they?

Reply:
Makes sense, but I'm not sure that we really need these difficulties because of this. Anyway, at first, it should be solved on `babel` side.

Reply:
So, it is not clear to me what should be changed (if anything).

Reply:
@zloirock is there an issue in Babel repo?

also for reference https://github.com/kangax/compat-table/issues/1217

Reply:
Hum, I see in #1217 that the issue is way more complicated than this patch.  I'll change both results to false and leave the comment until the tests are fixed.  
I'll keep the `var` change.  Is this OK?

Reply:
If it passes in the latest IE 11, it should be true.

Reply:
@chicoxyzzy seems it will be an issue when `babel` will update `compat-table` version.

Reply:
@afmenez as @ljharb proposed, you can add new positive results as a column for another IE11 build.

Reply:
Then babel can file a separate issue about it; the compat table doesn’t currently provide more than one set of results for each IE major, at “latest”.

Reply:
I believe adding another IE11 column would be confusing, as most users don't know about these "sub-versions".  I'll keep it as false with a note that both tests may pass later builds.

Reply:
Again, it should be true, not false.

Reply:
Again, it should be `false`, not `true` -) Just adding comments, like @afmenez wrote, should work for the first time, later we can add a new column for modern builds with positive results.

Reply:
The compat table has never listed non-latest IE builds - that is the standard we must follow.

Any bugs arising from that should be dealt with separately.

Reply:
> that is the standard we must follow

It was not a standard and I strictly disagree. See the reason above.

If someone wanna support IE he should support early IE builds because a serious part of users disabled auto updates -)

Reply:
That’s not the purpose of the compat table. IE 8, for example, only supports the JSON methods in a later build, but the table shows it as passing.

If you disagree that’s how it should work, that’s fine, but that disagreement is irrelevant to the fact that this is how the table operates, and that will not change in this unrelated PR.

Reply:
> That’s not the purpose of the compat table.

Really? Providing actual data about supported features is the main purpose of the compat table and we should not show a false positive -)

> IE 8, for example, only supports the JSON methods in a later build, but the table shows it as passing.

But the issue from this PR is known from 2014 and the table shows it as not passing. I do not care about the results of `JSON`. I don't see anything related your position in the documentation.

> If you disagree that’s how it should work, that’s fine, but that disagreement is irrelevant to the fact that this is how the table operates, and that will not change in this unrelated PR.

This PR just should not be added in this form.

Adding a new column with positive results makes sense.

Reply:
Updated the patch according to @ljharb orientation.

Reply:
Whether you care about the result of json or not is also irrelevant. It shows, clearly, that the table has never shown results from non-latest IE major versions. That shouldn’t change here. The bug is that this result sat at false for so long.

Reply:
One more time: if you wanna positive results for IE11 - feel free to add a new column. Changing results of the actual column will break `preset-env` and confuse developers who wanna support all IE11 versions. This precedent could cause much more serious problems in the future.

Reply:
The current column represents "IE 11 latest". If *you* want negative results for non-latest IE 11, then you're welcome to add a column.

Reply:
Me? -) Interesting position. The current column is just [IE11 with release date 2013-10-17](https://github.com/kangax/compat-table/blob/2c10d2d4fa6abee6aae21ef58fac299de6716249/environments.json#L393). It's definitely not "IE 11 latest".

Reply:
`preset-env` could override results. Also ideally it should use something what fits better for it than `compat-table`, which purpose never was to provide data for `preset-env` or anything else.

Reply:
> `preset-env` could override results.

It's additional special cases. Why?

> Also ideally it should use something what fits better for it than `compat-table`

Definitely. But currently we haven't any good alternative.

Reply:
a) the release date is for the first version, results always reflect the latest
b) supporting preset-env is ancillary and NOT something that this repo needs to cleave to, especially not when it changes the meaning this project's test columns have had historically.